src/PeriodOfStayInput.js

```diff
@@ -5,21 +5,39 @@
         DateInput = require('./DateInput'),
         Environment = require('./Environment'),
         Model = require('./Model'),
+        ReactIntl = require('react-intl'),
+        FormattedMessage = ReactIntl.FormattedMessage,
+        IntlMixin = ReactIntl.IntlMixin,
 
         nightsText = function (count) {
             switch (count) {
```

Comment:
Please get rid of that `switch (count)` completely, using a single ICU message for all the cases